pkg/ccl/multiregionccl/regional_by_row_test.go

```diff
@@ -799,6 +799,7 @@ USE t;
 	for _, rbrChange := range regionalByRowChanges {
 		for _, regionChange := range regionChanges {
 			t.Run(fmt.Sprintf("setup %s executing %s with racing %s", rbrChange.setup, rbrChange.cmd, regionChange.cmd), func(t *testing.T) {
+				defer log.Scope(t).Close(t)
```

Comment:
should this be after `defer leaktest.AfterTest(t)()` instead?

looks like the main_test.go is missing  `//go:generate ../../util/leaktest/add-leaktest.sh *_test.go` which i think (not sure after bazel) should lint against it. do you mind adding it?

Reply:
If I put it higher up, it would be a log scope for the whole outer test. Isn't it better to have one per subtest?

Reply:
Also, what do you mean about the leaktest lint? It's here: https://github.com/cockroachdb/cockroach/blob/dc9448130adee55cf34b72e38d76f2e4d6bc1c77/pkg/ccl/multiregionccl/main_test.go#L24

I don't think it says anything about log scopes.

Reply:
weird, thought it did.